AffirmSDK/AffirmRequest.m

```diff
@@ -181,7 +181,7 @@ - (NSDictionary *)parameters
         @"public_api_key": self.publicKey,
         @"user_confirmation_url": AFFIRM_CHECKOUT_CONFIRMATION_URL,
         @"user_cancel_url": AFFIRM_CHECKOUT_CANCELLATION_URL,
-        @"user_confirmation_url_action": @"POST",
+        @"user_confirmation_url_action": @"GET",
```

Comment:
What is the intention behind this change? Could this be parameterized from the call site instead?

Reply:
It's required by checkout api, and that's why it failed to get checkout token.